src/streamlink/__init__.py

```diff
@@ -1,17 +1,17 @@
 # coding: utf8
-"""Livestreamer extracts streams from various services.
+"""Streamlink extracts streams from various services.
 
-The main compontent of Livestreamer is a command-line utility that
+The main compontent of Streamlink is a command-line utility that
 launches the streams in a video player.
 
 An API is also provided that allows direct access to stream data.
 
-Full documentation is available at http://docs.livestreamer.io/.
+Full documentation is available at http://docs.streamlink.io/.
```

Comment:
We should probably modify this at some point to either use github pages, or read the docs to ensure it stays as openly accessible as possible.